dataflow-website/recipes/kinesisdemo/kinesisconsumer/pom.xml

```diff
@@ -15,7 +15,7 @@
 
 	<properties>
 		<java.version>1.8</java.version>
-		<spring-cloud.version>Greenwich.SR1</spring-cloud.version>
+		<spring-cloud.version>2020.0.4</spring-cloud.version>
```

Comment:
This is an older line of Spring Cloud, do you mind moving this up to `2021.0.5`.

Reply:
done.  :+1: